src/consts.h

```diff
@@ -82,10 +82,13 @@
 // PPS: 650 @ 5+1, 650 @ 5+3
 #define PACKET_BUNDLING_BUFFERED 2
 
-#define DEG_0 0.f
-#define DEG_90 -PI / 2
-#define DEG_180 PI
-#define DEG_270 PI / 2
+// Get radian for a given angle from 0° to 360° (2*PI*r, solve for r given an angle, range -180° to 180°)
+#define DEG_X(deg) (float)((((deg) < 180.0f ? 0 : 360.0f) - (deg)) * PI / 180.0f)
+
+#define DEG_0 DEG_X(0.0f)
+#define DEG_90 DEG_X(90.0f)
+#define DEG_180 DEG_X(180.0f)
+#define DEG_270 DEG_X(270.0f)
```

Comment:
```suggestion
#define DEG_0 0.f
#define DEG_90 -PI / 2
#define DEG_180 PI
#define DEG_270 PI / 2
```
The Values of the result are not the same. So it is preferable to stay with the existing formula. for the defined constant
```
[INFO ] [SensorManager]    DEG_0  : 0.00000000000000000000
[INFO ] [SensorManager] PI_DEG_0  : 0.00000000000000000000
[INFO ] [SensorManager]    DEG_90 : -1.57079637050628662109
[INFO ] [SensorManager] PI_DEG_90 : -1.57079632679489655800
[INFO ] [SensorManager]    DEG_180: 3.14159274101257324219
[INFO ] [SensorManager] PI_DEG_180: 3.14159265358979311600
[INFO ] [SensorManager]    DEG_270: 1.57079637050628662109
[INFO ] [SensorManager] PI_DEG_270: 1.57079632679489655800
```

Reply:
this is caused by comparing a double to a float.
the macro is cast as float, while the previous formula was automatically cast as double.
I'll fix the macro instead 😄

Reply:
Macro with cast to float
![image](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/assets/32248439/b331d65c-99c7-4900-9b45-4393c89b51a8)

Macro with cast to double
![image](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/assets/32248439/5e4834a5-24a5-4006-b2ab-a2609da9b113)